api/image/image_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package image
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsImageMatched(t *testing.T) {
+	testCases := map[string]struct {
```

Comment:
nit, I'd do these as a slice so the ordering isn't random

Reply:
I actually kinda prefer the map style since I find it more readable, but I've updated this to use slices.